client/index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+	<title>Chat prototype</title>
+	<script src="js/jquery-3.2.1.min.js"></script>
+	<script src="js/dataManager.js"></script>
+    <script src="https://unpkg.com/react@15/dist/react.js"></script>
+    <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
+    <script src="js/react/browser.min.js"></script>
+</head>
+<body>
+	<h1>
+		SjChat
+	</h1>
+	<script>
+		var apiAddress = window.location.protocol +'//'+ window.location.hostname +':8080';
+		var dataManager = new DataManager();
+
+		$.ajaxSetup({
```

Comment:
The lines seem to be necessary, when the lines are removed async is on. In fact, it complains that synchronous requests are deprecated.